test_conformance/allocations/allocation_execute.cpp

```diff
@@ -167,6 +167,10 @@ int execute_kernel(cl_context context, cl_command_queue *queue, cl_device_id dev
     access_string[0] = '\0';
     kernel_string[0] = '\0';
 
+    cl_uint *returned_results =
+        (cl_uint *)malloc(sizeof(cl_uint) * NUM_OF_WORK_ITEMS);
```

Comment:
This introduces a memory leak on the various error paths.  Suggesting to use a `std::vector` instead, so that no manual `free` is needed.

Reply:
Fixed, thanks.